Doc/tools/templates/indexcontent.html

```diff
@@ -1,5 +1,12 @@
-{% extends "defindex.html" %}
-{% block tables %}
+{% extends "layout.html" %}
+{%- block htmltitle -%}
+<title>{{ shorttitle }}</title>
+{%- endblock -%}
+{% block body %}
+  <h1>{{ docstitle|e }}</h1>
+  <p>
+  {% trans %}Welcome! This is the documentation for Python {{ release }}{% endtrans %}
```

Comment:
Sorry, I just noticed that there is no dot at the end of the sentence. It's also missing in #165 and other backport PRs.